Sources/Plugins/ExternalSensorsPlugin/BLE/DeviceSettings/DevicesSettingsCollection.swift

```diff
@@ -79,6 +78,20 @@ final class DevicesSettingsCollection {
         }
     }
     
+    func changeWheelSize(with id: String, size: Float) {
```

Comment:
1. There should be more common method like `changeDeviceParameter` (device id, param id, value).
2. Use `setDeviceProperty` for settings parameter